staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -36,12 +36,12 @@ const (
 
 	// SeccompPodAnnotationKey represents the key of a seccomp profile applied
 	// to all containers of a pod.
-	// Deprecated: set a pod security context `seccompProfile` field.
+	// Deprecated: set a pod security context `seccompProfile` field. The key is non functional in 1.25+.
 	SeccompPodAnnotationKey string = "seccomp.security.alpha.kubernetes.io/pod"
 
 	// SeccompContainerAnnotationKeyPrefix represents the key of a seccomp profile applied
 	// to one container of a pod.
-	// Deprecated: set a container security context `seccompProfile` field.
+	// Deprecated: set a container security context `seccompProfile` field. The key is non functional in 1.25+.
```

Comment:
Is this comment still accurate? It looks like some code does read the annotations.

https://github.com/kubernetes/kubernetes/pull/109819/files#r880206105

> I added a patch … for copying the annotations to the fields (and not the other way around).

Reply:
No you're right. I removed the additional comment because I think the exact behavioral description should be part of the official docs.